dashboard/test/utils/fake_google_account.dart

```diff
@@ -31,8 +31,8 @@ class FakeGoogleSignInAccount implements GoogleSignInAccount {
   Future<void> clearAuthCache() => Future<void>.value(null);
 
   @override
-  bool operator ==(dynamic other) {
-    final GoogleSignInAccount otherAccount = other;
+  bool operator ==(Object other) {
+    final GoogleSignInAccount otherAccount = other as GoogleSignInAccount;
```

Comment:
There is some technical debt here, as this cast could fail at runtime. But I am only here to make a no-op change, and not change behavior.

Reply:
umm could Sam elaborate a bit more on the "fail at runtime" part

Reply:
If `other` is not a `GoogleSignInAccount`, then this code fails at runtime (both before this PR and with this PR).

Reply:
ahh sounds good to me. I think we are good to merge then!